portalnet/src/utils/db.rs

```diff
@@ -66,7 +66,7 @@ pub fn configure_node_data_dir(
 fn get_node_data_dir(trin_data_dir: PathBuf, node_id: NodeId) -> PathBuf {
     // Append first 8 characters of Node ID
     let mut application_string = "trin_".to_owned();
-    let node_id_string = hex_encode(node_id.raw());
+    let node_id_string = hex::encode(node_id.raw());
```

Comment:
Sorry, I'm late to this review. But it seems like we can avoid an unnecessary library import.. `hex_encode` is an internal utility fn that always returns a `0x` prefixed string. I believe there are multiple possible solutions to removing the `0x` prefix that don't require importing the `hex` library.
eg. `node_id_string.trim_start_matches("0x")` or even something like `let suffix = &node_id_string[2..8]` I believe would work. This is just a suggestion, but imo, if we can avoid unnecessary library imports then it's worth considering.